tests/library/gemm_test.py

```diff
@@ -83,10 +84,9 @@ def run_test(implementation,
         C_shape = [s if isinstance(s, int) else replace_map[s] for s in C_shape]
 
     # unique name for sdfg
-    C_str = "None" if C_shape is None else (
-        str(C_shape[0]) if len(C_shape) == 1 else f"{C_shape[0]}_{C_shape[1]}")
-    sdfg_name = f"{implementation}_{M}_{N}_{K}_{complex}_{transA}_{transB}_{alpha}_{beta}_{C_str}".replace(
-        ".", "_dot_").replace("+", "_plus_").replace("(", "").replace(")", "")
+    sdfg_name = random.choices(string.ascii_lowercase)[0]
+    sdfg_name += ''.join(random.choices(string.ascii_lowercase + string.digits,
+                         k=19))
```

Comment:
Haha now it's even crazier! If you just want a mangled name, why not use `Config.set("cache", "unique")` or `Config.set("cache", "hash")`? You can still get the build folder with `sdfg.build_folder` if you need to